cartopy/documentation.html

```diff
@@ -53,14 +54,32 @@
           <h2>Previous releases</h2>
             <ul>
                 <li><a href="docs/v0.13/index.html">v0.13</a></li>
+            </ul>
```

Comment:
Less facetiously, it is a bit odd. If you don't like it I'll change it (I'm not a huge fan)